CSharp/BatchExplorer/Models/TaskModel.cs

```diff
@@ -285,6 +327,39 @@ public override List<PropertyModel> PropertyModel
 
                     await asyncListSubtasksTask;
 
+                    // Linked Storage Account Files
```

Comment:
I didn't know about TaskOutputStorage.ListOutputs() but it doesn't seem quite useful w/o showing all kinds. Yes, I want to list all outputs under the task, regardless of kind.